control/state.py

```diff
@@ -162,6 +162,8 @@ class OmapGatewayState(GatewayState):
     """
 
     OMAP_VERSION_KEY = "omap_version"
+    OMAP_FILE_LOCK_NAME = "omap_file_lock"
+    OMAP_FILE_LOCK_COOKIE = ""
```

Comment:
Why is the lock cookie value empty?

Reply:
@baum we don't want different gateways updating the OMAP at the same time so they should all use the same lock name and cookie. We might issue a different cookie (or lock name) in case the OMAP file name is different. I will look into it.